supply/lib/supply/uploader.rb

```diff
@@ -81,6 +85,21 @@ def perform_upload_meta(version_codes, track_name)
           version_codes.to_s == ""
         end
 
+        upload_worker = FastlaneCore::QueueWorker.new do |job|
+          begin
```

Comment:
I _think_ `begin`…`end` is unnecessary when they are encapsulating the whole body of a block? i.e. I think you can remove `begin` and `end` while still keeping your `rescue` here?

Reply:
good catch! will change